packages/next/client/image.tsx

```diff
@@ -560,6 +561,68 @@ function cloudinaryLoader({ root, src, width, quality }: LoaderProps): string {
   return `${root}${paramsString}${normalizeSrc(src)}`
 }
 
+function uploadcareLoader({ root, src, width, quality }: LoaderProps): string {
+  const isOnCdn = /^https?:\/\/ucarecdn\.com/.test(src)
```

Comment:
what if i use custom cname like images.example.com for my uploadcare images?

Reply:
That's why we need ability to pass custom options directly to the image loader. I don't want to change Image Loader API without discussion with Next.js team.